tools/tpm2_getmanufec.c

```diff
@@ -407,22 +409,16 @@ int RetrieveEndorsementCredentials(char *b64h)
     size_t len = 1 + strlen(b64h) + strlen(EKserverAddr);
     char *weblink = (char *)malloc(len);
     if (!weblink) {
-        LOG_ERR("Could not open file for writing: \"%s\"", ECcertFile);
-                return ret;
+        LOG_ERR("oom");
+        return ret;
```

Comment:
LOG_ERR has filename and number, so a simple oom is  enough, considering the other data printed.